code-server/run.sh

```diff
@@ -104,7 +104,8 @@ if [ "${AUTO_INSTALL_EXTENSIONS}" = true ]; then
 
   if [ -f "$WORKSPACE_DIR/.vscode/extensions.json" ]; then
     printf "🧩 Installing extensions from %s/.vscode/extensions.json...\n" "$WORKSPACE_DIR"
-    extensions=$(jq -r '.recommendations[]' "$WORKSPACE_DIR"/.vscode/extensions.json)
+    # Use sed to remove single-line comments before parsing with jq
+    extensions=$(sed 's|//.*||g' "$WORKSPACE_DIR"/.vscode/extensions.json | jq -r '.recommendations[]')
```

Comment:
This is sufficient as a quick fix. However, since both scripts will have Node available, we may want to consider parsing and stringifying the string using Node, as that should also eliminate multiline comments and comments after a field or value. (or even use a third-party NPM package to do so).

Reply:
That would be great. We can also get rid of `jq` then.